cli/pazuzu/actions/search.go

```diff
@@ -0,0 +1,44 @@
+package actions
+
+import (
+	"errors"
+	"fmt"
+	"github.com/urfave/cli"
+	"github.com/zalando-incubator/pazuzu/config"
+	"os"
+	"regexp"
+	"text/tabwriter"
+)
+
+func Search(c *cli.Context) error {
+	if c.NArg() != 1 {
+		return errors.New("Search regexp is not provided")
+	}
+	feature := c.Args().First()
+	featureRegexp, err := regexp.Compile(feature)
```

Comment:
On the second thought there will be a lot of type conversion when we get value back from this function.